src/path.cpp

```diff
@@ -168,6 +168,15 @@ wcstring_list_t path_get_paths(const wcstring &cmd, const environment_t &vars) {
     return paths;
 }
 
+std::vector<std::string> path_get_paths_ffi(const wcstring &cmd, const parser_t &parser) {
```

Comment:
With the new `wcstring_list_ffi_t` this can be:

    wcstring_list_ffi_t path_get_paths_ffi(const wcstring &cmd, const parser_t &parser) {
         return path_get_paths(cmd, parser);
    }
    
Then on the Rust side:

    paths: Vec<WString> = ffi:: path_get_paths_ffi(cmd, parser).from_ffi();
    
pretty nice

Reply:
Oooh, nice.

Done in three commits so I can make them disappear.